public/v1/components/schemas.yaml

```diff
@@ -1191,7 +1194,9 @@ components:
         - $ref: 'schemas.yaml#/components/schemas/BaseTestStatus'
         - const: offline
     TimingPacketRtt:
-      type: number
+      type:
+        - number
+        - 'null'
```

Comment:
So I think this one can be reverted?

Reply:
Probe fix should be released first

Reply:
Oh, ok, this actually brings me to one of the questions I had and forgot to ask. If we depend on the released probe image here, it doesn't allow us to test anything before a release and it seems that'll generally be a problem when adding features that modify both repos. It seems it would be better to run these tests with the probe master branch? We could either add a build for that via actions or pull the repo here and build the image locally.

Reply:
> Later, when there are changes in both repos, and both are in feature branches they will block each other from merging (kind of deadlock). To fix that we may try to pull repo from the branch with the same name first (e.g. e2e test in api branch feature-1 will try to pull feature-1 from probe, if it doesn't exist, pull master. Then build image from sources and run tests.).

That's a scenario I thought about a lot, and maybe it isn't really an issue because we need backward compatibility as well, so the following might work:
1. Make PR in the API repo. Write tests so that they pass regardless of probe version.
2. Merge API PR.
3. Make PR in the probe repo (or restart CI if already open). Tests pull the latest API here.
4. Merge probe PR.

This way we test both new and old probes with each API, which seems like a good thing.

Reply:
Sounds valid, I'll update the pull behaviour then.